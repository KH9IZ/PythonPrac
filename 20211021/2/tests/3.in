:457157 s s.capitalize()
457157 "Точно, пароль же есть на email, совсем забыл. Восстановление не потребуется)Спасибо"
:IlidanNaga c c * 517 + 100500
IlidanNaga 2021
IlidanNaga 963
457157 "fAFAFAF"
quit https://t.me/uneex_talks
