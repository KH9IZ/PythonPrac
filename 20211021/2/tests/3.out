Точно, пароль же есть на email, совсем забыл. восстановление не потребуется)спасибо
1145357
598371
Fafafaf
3
